ledger/catchpointwriter.go

```diff
@@ -240,7 +266,7 @@ func (cw *catchpointWriter) asyncWriter(balances chan catchpointFileBalancesChun
 		if chunkLen := uint64(len(encodedChunk)); cw.biggestChunkLen < chunkLen {
 			cw.biggestChunkLen = chunkLen
 		}
-		if numAccountsProcessed == cw.totalAccounts {
+		if numAccountsProcessed == cw.totalAccounts { // Quits too soon. Consider KVs
```

Comment:
@jannotti Sanity checking - My comprehension is weak here, but the inline comment makes me feel like there's more changes intended.  Is that the case, is the comment stale, or something else?

In case it's relevant, there's a similar condition on line 219 (`cw.numAccountsProcessed == cw.totalAccounts`).  If a change is warranted here, perhaps a change is needed there too?

Reply:
I put that there during merge to remind me that I wasn't done.  I'll be committed a reasonable candidate momentarily.